draft-tschofenig-rats-psa-token.md

```diff
@@ -608,6 +635,86 @@ A PSA token is always directly signed by the PSA RoT.  Therefore, a PSA
 claims-set ({{sec-psa-claims}}) is never carried in a Detached EAT bundle
 ({{Section 5 of EAT}}).
 
+### Freshness Model
+
+The PSA Token supports the freshness models for attestation Evidence based on
+nonces and epoch handles ({{Section 10.2 and Section 10.3 of RFC9334}}) using
+the `nonce` claim to convey the nonce or epoch handle supplied by the Verifier.
+No further assumption on the specific remote attestation protocol is made.
+
+Note that use of epoch handles is constrained by the type restrictions imposed by the `eat_nonce` syntax.
+For use in PSA tokens, it must be possible to encode the epoch handle as an opaque binary string between 8 and 64 octets.
+
+### Synopsis
+
+{{tbl-baseline-profile}} presents a concise view of the requirements.
+
+| Issue | Profile Definition |
+| CBOR/JSON | CBOR MUST be used  |
+| CBOR Encoding | Definite length maps and arrays MUST be used |
+| CBOR Encoding | Definite length strings MUST be used |
+| CBOR Serialization | Variant serialization MAY be used |
+| COSE Protection | COSE_Sign1 and/or COSE_Mac0 MUST be used |
+| Algorithms | {{COSE-ALGS}} SHOULD be used |
+| Detached EAT Bundle Usage | Detached EAT bundles MUST not be sent |
+| Verification Key Identification | Any identification method listed in {{Appendix F.1 of EAT}} |
+| Endorsements | See {{sec-psa-endorsements}} |
+| Freshness | nonce or epoch ID based |
+| Claims | Those defined in {{sec-psa-claims}}. As per general EAT rules, the receiver MUST NOT error out on claims it doesn't understand. |
+{: #tbl-baseline-profile title="Baseline Profile"}
+
+## Profile LM
```

Comment:
I suggested changing to `## Profile LM (Large / Medium)` as this appears in the table of content.